.circleci/config.yml

```diff
@@ -1017,13 +1017,15 @@ workflows:
       - build-linux-arm-ubuntu:
           <<: *never
       - build-platforms:
-          <<: *on-integ-and-version-tags
+          #@@<<: *on-integ-and-version-tags
+          <<: *always
           context: common
           matrix:
             parameters:
               platform: [jammy, focal, bionic, centos7, rocky8, bullseye, amzn2]
       - build-arm-platforms:
-          <<: *on-integ-and-version-tags
+          ##@<<: *on-integ-and-version-tags
+          <<: *always
```

Comment:
always?

Reply:
this is diagnostic - I removed it after everything relevant passed.